backend/src/test/java/com/woowacourse/kkogkkog/application/CouponServiceTest.java

```diff
@@ -167,20 +176,31 @@ class ChangeStatusTest {
         class Ready {
 
             @Test
-            @DisplayName("받은 사람이 REQUEST 를 보내면, 쿠폰의 상태를 REQUESTED 로 변경한다.")
+            @DisplayName("받은 사람이 REQUEST 와 약속날짜를 보내면, 쿠폰의 상태를 REQUESTED 로 변경한다.")
             void success_request() {
```

Comment:
문구를 전반적으로 이런 느낌으로 수정하는게 가독성이 높지 않나 싶어요.

```suggestion
            @DisplayName("받은 사람이 REQUEST 이벤트와 약속날짜를 보내면, 쿠폰을 REQUESTED 상태로 변경한다.")
            void success_request() {
```